### PR TITLE
Following feedback in #259

### PR DESCRIPTION
**Pull request by webmasterkai**
See issue #259.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ ReactDOM.render(
 )
 ```
 
-Now you can read from `state.routing.location.pathname` to get the URL. It's far more likely that you want to change the URL more often, however. You can use the `push` action creator that we provide:
+Now you can read from `state.routing.locationBeforeTransitions.pathname` to get the URL. It's far more likely that you want to change the URL more often, however. You can use the `push` action creator that we provide:
```
Comment (timdorr) [inline README.md]: User are discourage from reading state at all. I'm thinking of prefixing this key with a `$` to make that clear. But this section of the README should actually go away or be significantly changed.

**Comment by timdorr** [comment]
Cool, I'll merge this back into the branch so it shows up in the PR for further work.

File: README.md (hunk 1 above)
Comment (webmasterkai) [inline README.md]: What's the solution for users that refuse to define routes in jsx within a react component? Where should a user parse the location so they can get things like named path parameters using a tool like url-pattern?

Comment (timdorr) [inline README.md]: Router hasn't had named routes for a while now. You should be using URLs or location descriptors for all of your navigation.

Comment (webmasterkai) [inline README.md]: I'd like to make a simple example that does not use react-router. How do I gain access to the location.pathname?

Comment (timdorr) [inline README.md]: Listen to the history. It's a synchronous operation.

Comment (webmasterkai) [inline README.md]: Isn't that the point of this library? To listen to history and put it in the same place where every other component reads state from?

**Pull request merged**